hpx/runtime/serialization/vector.hpp

```diff
@@ -23,13 +23,11 @@ namespace hpx { namespace serialization
         ar >> size; //-V128
         if(size == 0) return;
 
-        vs.reserve(size);
+        vs.resize(size);
```

Comment:
resize() causes the default c-tors of all objects in the vector to run, which can be costly. I don't see how this would lead to a speed-up.